main.tf

```diff
@@ -527,31 +529,61 @@ data "aws_iam_policy_document" "combined" {
 }
 
 # AWS Load Balancer access log delivery policy
-data "aws_elb_service_account" "this" {
-  count = local.create_bucket && var.attach_elb_log_delivery_policy ? 1 : 0
+locals {
+  # List of AWS regions where permissions should be granted to the specified Elastic Load Balancing account ID ( https://docs.aws.amazon.com/elasticloadbalancing/latest/application/enable-access-logging.html#attach-bucket-policy )
+  elb_service_accounts = {
```

Comment:
This list of regions won't change in the future, so we can hardcode them here. Also, `aws_elb_service_account` data-source doesn't work for newer regions.